### ISSUE TITLE
graphQL - stateQuery>avatar>inventory>equipments>stat>type

### ISSUE THREAD

**Issue by MiningBurn** [planetarium/NineChronicles.Headless#358]
receive error when querying type. Think it should return a string from the enum StatType
```graphql
"errors": [
    {
      "message": "Error trying to resolve field 'type'.",
      "locations": [
        {
          "line": 47,
          "column": 13
        }
      ],
      "path": [
        "stateQuery",
        "avatar",
        "inventory",
        "equipments",
        0,
        "stat",
        "type"
      ],
 ```

**Comment by moreal** [comment]
Hello @MiningBurn. Could you attach the query you used, please?

**Comment by MiningBurn** [comment]
query
{
 stateQuery{
  avatar(address:"0x6F57DdD8dA79897bcC9Aa7EE79749BFFD106B8e9"){
    inventory{  
        equipments{
	      elementalType
 				equipped
 				grade
 				id
  			itemId
  			itemSubType
  			itemType
  			setId
  			stat{
                               type
                               value}
    }
   }
  }
 }
}

**Comment by MiningBurn** [comment]
If I comment out type the query works fine.

**Comment by moreal** [comment]
@MiningBurn I looked your query. I found the bug and it will be fixed in #363. It may be contained in the next release. Thanks to report the bug! 🙏 

![image](https://user-images.githubusercontent.com/26626194/110905364-b528c300-834d-11eb-91ff-68e4ee3165d0.png)
